### PR TITLE
tint remote (ZBT-Remote) Color keys

### PR DESCRIPTION
**Pull request by MoskitoHorst**

### CODE REVIEW

**Comment by Koenkk** [comment]
thanks, could you rebase this pr?

**Comment by MoskitoHorst** [comment]
rebased the pull request, now all action-results are key names, added Parameter values, even if they seem constant

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -1564,22 +1564,43 @@ const converters = {
...
-            return {action: 'toggle'};
+            return {action: 'power', state: 'on'};
         },
     },
     tint404011_off: {
         cid: 'genOnOff',
         type: 'cmdOff',
         convert: (model, msg, publish, options) => {
-            return {action: 'toggle'};
+            return {action: 'power', state: 'off'};
```
Comment (Koenkk) [inline converters/fromZigbee.js]: Why not just return `action: 'off`' here (same for on)

Comment (MoskitoHorst) [reply converters/fromZigbee.js]: Therefore was noch reasonable, wenn can so ist with in anderen Off.

Comment (Koenkk) [reply converters/fromZigbee.js]: I don't understand, could you explain this further?

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**